src/layout/section/illustratedSection/illustratedSection.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import BaseSection, { SectionContentSize } from 'layout/section/baseSection'
+
+export interface IllustratedSectionProps {
+  readonly children: React.ReactNode
+  readonly className?: Classcat.Class
+  readonly illustrationUrl: string
+  readonly illustrationAlt?: string
+}
+
+/**
+ * A specialized section which show some content with an illustration.
+ */
+const IllustratedSection: React.SFC<IllustratedSectionProps> = (props: IllustratedSectionProps) => {
+  const { className, illustrationUrl, illustrationAlt = '' } = props
+  return (
+    <BaseSection tagName="article" className={className} contentSize={SectionContentSize.LARGE}>
+      <div className="kirk-illustratedSection-illustration">
+        <img src={illustrationUrl} alt={illustrationAlt} />
```

Comment:
Warning: in the Profile picture step, we're not displaying an illustration, but an `Avatar`. How do you think we can handle this?

Reply:
Maybe juste have a `illustrationMedia=string | JSX.Element ` and do different rendering based on type ? And pray to not have more rendering cases :)

Reply:
Good point for the Profile Picture case. I'll add support for the Avatar component :)